lib/steps.js

```diff
@@ -227,6 +227,45 @@ When(/^(?:the user gets)?(?:I get)? to ?(?:the)?(?: ?question)?(?: id)? "([^"]+)
 
   }  // ends while !id_matches
   // Can anything unsatisfactory get through here?
+
+  // Add report strings from supported table, not an altered one
+  for ( let row of supported_table ) {
+    row.report_str = `${ await scope.convertToOriginalStoryTableRow(scope, { row_data: row }) }`;
+  }
+  // Sort into aphabetical order, as is recommended for writing story tables
```

Comment:
I understand that alphabetical is how the story tables are recommended to be written, but I don't think that's useful for looking at a test report: the rows should be chronological, so it matches the overall chronological structure of the report (and matches the previously shown screen ids).

Reply:
I hear that and thank you for the feedback!

Regarding order of rows, I didn't quite get to go into that during the discussion, so I'll try to do so here.

First, and least significant: the generator generates a bunch of extra rows because it doesn't know how to tell which variables are or aren't necessary. One hope for this was that it could help people who are using the generator so that they can simplify their table.

Second, and more compelling for me: When you're looking for a specific row to see where it appears, which is what I imagine these tables being useful for, it's hard to remember which row appeared where in chronological order, especially if you're changing the order of screens a lot as some of our developers do. If the rows are in alphabetical order, it would be easier to locate any one specific row. It's the same reason it's recommended for the tables in general. I've found it useful, at least. It probably depends on the type of problems one runs into the most.

I think I had another reason, but I can't recall right now.

Reply:
Also, not exactly sure yet how to handle catching errors in this environment where steps are fragmented in such a way. Would love some eyes on that.

Reply:
Made a new PR for an experiment on testing failing tests. I _think_ it's what we're looking for.